test/TestUtilities/Test.Utility/TestDirectoryBuilder.cs

```diff
@@ -67,6 +67,8 @@ public TestDirectory Build()
 
             if (NuspecBuilder != null)
             {
+                var dir = Path.GetDirectoryName(NuspecPath);
+                Directory.CreateDirectory(dir);
```

Comment:
I don't see it in other places, either, but shouldn't these directories get deleted? Or we just rely on scripts to do that later?

Reply:
The directories are deleted once `TestDirectory` is deleted. That class have a `Dispose()` method.